bin/key_server.js

```diff
@@ -99,7 +99,6 @@ function run(config) {
               serverPublicKeys,
               signer,
               db,
-              bounces,
```

Comment:
Not sure when it happened, but I couldn't find any usage of `bounces` in the routes so stopped passing it in completely.